### PR TITLE
Create `firefox/campaign.lang` file

### PR DESCRIPTION
**Pull request by alexgibson**
## What does this PR do?

- Creates a new `firefox/campaign.lang` file, copying the existing strings from `firefox/new/quantum.lang`.

## Why do we need this?

- Marketing want to create a separate version of `/firefox/new/` at the URL `/firefox/campaign/` for targeted Ad traffic. The expectation is that this new page will operate completely independently from whatever changes on /new, hence the need for a separate lang file.
- Related bedrock PR https://github.com/mozilla/bedrock/pull/6873
- Demo URL: https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/campaign/

## What doesn't this PR do?

- Right now this is just a straight up copy & renaming of the existing lang file. I haven't gone as far as removing any unneeded strings to try and keep things simple for review. But I'm happy to try and tidy things up further, if that may be a worthwhile thing to spend some time on.

### CODE REVIEW

**Comment by alexgibson** [comment]
@peiying2 r?

**Comment by peiying2** [comment]
@alexgibson, the PR itself is most likely clean, but it can’t go out like this. There are a few steps involved so the new files are added to github on l10n mozorg repo, Pontoon, with defined locales to support, deadline, priority and the number of strings.

**Comment by peiying2** [comment]
This might be a good project for us to talk about the string extraction training and understand the whole l10n flow to get the files ready

**Comment by alexgibson** [comment]
> This might be a good project for us to talk about the string extraction training and understand the whole l10n flow to get the files ready

I would love for that training to happen, but as always there's some urgency with this request as it's blocking other work 😞

Please tell me what I need to do, or show me where to go.

**Comment by peiying2** [comment]
There is a process flow doc that is being reviewed and updated and we will share with you soon. 

@flodolo given where this PR is, can I update the langchecker part to make it work?

**Comment by flodolo** [comment]
> @flodolo given where this PR is, can I update the langchecker part to make it work?

Let me do it. I want to double check that everything looks good and there are no errors locally, and it's going to be painful with your connection.

@alexgibson 
TL;DR. All files that we have in mozilla.org are tracked centrally by langchecker. We only manually set up en-US, then use scripts to propagate changes, and do all other things (activate, add tags, etc.)

**Comment by flodolo** [comment]
>     * But I'm happy to try and tidy things up further, if that may be a worthwhile thing to spend some time on.

That would be great, even after this PR. The reason is that locales will have duplicate strings, translated or missing, to look at, when they're actually not used.

Just remove the strings from en-US file, then we'll run scripts to update other locales.

**Comment by alexgibson** [comment]
Thanks both!

> Just remove the strings from en-US file, then we'll run scripts to update other locales.

I'll open a new PR to remove the unneeded strings from the en-US file 👍

**Comment by alexgibson** [comment]
PR to clean up strings: https://github.com/mozilla-l10n/www.mozilla.org/pull/278
